src/lib/driver/command.ts

```diff
@@ -1,6 +1,8 @@
 export enum DriverCommand {
+  getConfig = "driver.get_config",
   updateLogConfig = "driver.update_log_config",
   getLogConfig = "driver.get_log_config",
   enableStatistics = "driver.enable_statistics",
   disableStatistics = "driver.disable_statistics",
+  statisticsEnabled = "driver.statistics_enabled",
```

Comment:
can you add a word to describe what is done with `statistics_enabled`, maybe `is_statistics_enabled` ?